InAppUtils/InAppUtils.m

```diff
@@ -189,6 +189,37 @@ - (void)dealloc
     [[SKPaymentQueue defaultQueue] removeTransactionObserver:self];
 }
 
+RCT_EXPORT_METHOD(paidForApp:(RCTResponseSenderBlock)callback)
```

Comment:
This should be called `restorePurchase` and if it returns with some results - it was purchased. In fact every app according to apple guidelines needs that button.